README.md

```diff
@@ -5,53 +5,39 @@
 # Atmospheric moisture tracking
 
 This repository contains the source code of the WAM2layers moisture tracking
-code originally developed by Ruud van der Ent. It can be used to determine where
+code. This code is currently developed by a core team:
+Ruud van der Ent (Delft University of Technology)
+Imme Benedict (Wageningen University)
+Chris Weijenborg (Wageningen University)
+Peter Kalverla (Netherlands eScienceCenter)
+
+It can be used to determine where
 precipitation originally evaporated (backtracking), or where evaporated moisture
 eventually ends up (forward tracking).
 
-See the [documentation](http://wam2layers.rtfd.io/) for a more detailed description.
-
 # How to use
-
-```
-# Advised but not strictly needed
-conda create -n wam2layers -c conda-forge python=3.9 jupyterlab cartopy matplotlib
-conda activate wam2layers
-
-# Simple user installation
-pip install git@github.com:WAM2layers/WAM2layers.git
-
-
-# Editable (develop) installation
-git clone git@github.com:WAM2layers/WAM2layers.git
-cd WAM2layers/
-pip install -e .[develop]
-
-# Download sample data and update paths in config file
-https://rebrand.ly/wam2layers-example-data
-
-# Run the example backtrack script
-wam2layers backtrack floodCase_202107.yaml
-wam2layers visualize output floodCase_202107.yaml
-```
+See the [documentation](http://wam2layers.rtfd.io/) for a more detailed description.
```

Comment:
I would like to keep a short overview in the GH landing page as well. Typically that's where developers end up, and it is useful for them to have a quick idea of the capabilities of the software. We can simplify it a bit though and indeed add a prominent pointer to the more extensive documentation.